### PR TITLE
Aarch Migration - again

### PR DESCRIPTION
**Pull request by sodre**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Close #71 
Close #73

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

**Comment by sodre** [comment]
@hmaarrfk, I must have missed something, how can I get the other architecture .ci support included in repo?

File: conda-forge.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
```
Comment (hmaarrfk) [inline conda-forge.yml:6]:
```suggestion
provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
```

**Comment by hmaarrfk** [comment]
maybe draft is causing problems.

maybe just use azure?

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
```
Comment (hmaarrfk) [inline conda-forge.yml:6]:
```suggestion
provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
```

Comment (hmaarrfk) [reply conda-forge.yml:6]: ??? why didn't rerendering work...

**Comment by hmaarrfk** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
can you remove the draft marking?

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml (hunk 2 above)
Comment (sodre) [reply conda-forge.yml:6]: I had an extra "provider" line at the bottom of the conda-forge.yml file that was clobbering the one you put in.

**Comment by sodre** [comment]
The bad news, we need to find a way to bootstrap golang on aarch64 and ppc64le. 

Do you have any resources/links/things to buy/vm/amis so I can do some of the tests locally first?

**Comment by sodre** [comment]
@hmaarrfk, this is not looking very promising. Our best bet might be to repackage the binaries from upstream.

If you think that is viable short-term, them we need to merge conda-forge/staged-recipes#9535

**Pull request closed without merging**